plugins/forward-webhook/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@flatfile/plugin-forward-webhook",
```

Comment:
I'm afraid this is going to get confused with the `@flatfile/plugin-webhook-egress` plugin. I'm thinking we call this `@flatfile/plugin-webhook-event-forwarder`.

Reply:
Good catch, I'll update that